ui/narrative/methods/import_tsv_as_phenotype_set_from_staging/display.yaml

```diff
@@ -21,24 +21,21 @@ suggestions :
 
 parameters :
     staging_file_subdir_path :
-        ui-name : |
-            Phenotype TSV File Path
+        ui-name : Phenotype TSV file path
         short-hint : |
             Select a phenotype file in TSV format
         long-hint  : |
             Filename should end in .tsv
 
     phenotype_set_name :
-        ui-name : |
-            Phenotype Set Name
+        ui-name : Phenotype Set name
```

Comment:
Set?

Reply:
it isn't just a set, it's a Phenotype Set!